src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started to work");

```

Comment:
The message 'Truck started to work' should be more informative and indicate that it is a 'Truck' specifically that is working. Consider rephrasing it to include the type of machine for clarity, as per the checklist's guidance on writing informative messages in methods.